eis_qgis_plugin/eis_processing/algorithms/prediction/weights_of_evidence_calculate_responses.py

```diff
@@ -0,0 +1,77 @@
+from qgis.core import (
+    QgsProcessing,
+    QgsProcessingParameterMapLayer,
+    QgsProcessingParameterMultipleLayers,
+    QgsProcessingParameterRasterDestination,
+)
+
+from eis_qgis_plugin.eis_processing.eis_processing_algorithm import EISProcessingAlgorithm
+
+
+class EISWeightsOfEvidenceCalculateResponses(EISProcessingAlgorithm):
+    def __init__(self) -> None:
+        super().__init__()
+
+        self._name = "weights_of_evidence_calculate_responses"
+        self._display_name = "Weights of evidence calculate responses"
+        self._group = "Prediction"
+        self._group_id = "prediction"
+        self._short_help_string = "Calculate the posterior probabilities for the given generalized weight rasters."
+
+        self.multiple_layers_as_typer_option = True
+
+    def initAlgorithm(self, config=None):
+        self.alg_parameters = [
+            "input_rasters_weights",
+            "input_rasters_standard_deviations",
+            "input_weights_table",
+            "output_probabilities",
+            "output_probabilities_std",
+            "output_confidence_array"
+        ]
+
+        input_rasters_weights_param = QgsProcessingParameterMultipleLayers(
+            name=self.alg_parameters[0], description="Weight rasters", layerType=QgsProcessing.TypeRaster
+        )
+        input_rasters_weights_param.setHelp(
+            "Output weight rasters (W+/Generalized W+) of the weights of evidence calculate weights tool."
+        )
+        self.addParameter(input_rasters_weights_param)
+
+        input_rasters_std_param = QgsProcessingParameterMultipleLayers(
+            name=self.alg_parameters[1],
+            description="Weight standard deviation rasters",
+            layerType=QgsProcessing.TypeRaster,
+        )
+        input_rasters_std_param.setHelp("""
+            Output weight standard deviation rasters (S_W+/Generalized S_W+) of the weights of evidence calculate
+            weights tool."""
+        )
+        self.addParameter(input_rasters_std_param)
+
+        input_weights_file = QgsProcessingParameterMapLayer(
+            name=self.alg_parameters[2], description="Results table"
+        )
```

Comment:
This could be better so that we don't let the user select raster layers. Didn't find a way to restrict to only no-geometry vector layers unfortunately. Also, could add to the parameter description that the table needs to include deposit and `Deposit count` and `Pixel count` columns
```suggestion
        input_weights_file = QgsProcessingParameterVectorLayer(
            name=self.alg_parameters[2], description="Results table", types=[QgsProcessing.SourceType.TypeVector] 
        )
```